bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1567,13 +1598,6 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 </finidat>
 
 <!-- Present day for coupled cases at 1-degree or lower resolutions  -->
-<!-- NB: This pSASU finidat is from an 1850 spin-up -->
```

Comment:
Was this removed because it was a duplicate?

Reply:
Looks like there might be more removed below for the same reason. We do need to make sure the different physics options are covered. Although maybe since we are removing clm5_1, we should drop adding it in...

Reply:
Yes, all the removed ones were duplicates. I was careful to confirm, and I hope I didn't miss something.

Reply:
I looked again and confirmed, so I'm resolving this one.